GirafRest/Controllers/AccountController.cs

```diff
@@ -52,6 +53,11 @@ public class AccountController : Controller
             IOptions<JwtConfig> configuration,
             IAuthenticationService authentication)
         {
+            if (giraf == null) {
+                throw new System.ArgumentNullException(giraf + " is null");

```

Comment:
I praksis skriver du her `null is null` i warning. Du skal have nameof på argumentet i stedet.